aio/content/examples/rx-library/src/simple-creation.1.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { docRegionPromise } from './simple-creation.1';
+import { of } from 'rxjs';
+
+describe('simple-creation.1', () => {
+  it('should create a promise from an observable and return an empty object', () => {
+    const console = {log: jasmine.createSpy('log')};
+    const fetch = () => of({});
```

Comment:
Let's use a property to make this more distinct:

```suggestion
    const fetch = () => of({foo: 42});
```